site-packages/integralstor_utils/config.py

```diff
@@ -115,6 +117,18 @@ def get_minimum_user_uid_gid(type='user'):
         return id, None
 
 
+def get_normal_users():
+    try:
+        ret, err = command.get_command_output("awk -F'[/:]' '{if ($3 >= 1000 && $3 < 65534) print $1}' /etc/passwd")
```

Comment:
I did consider that and discarded it because is set as 1500 in `get_minimum_user_uid_gid()`. So, the users that we create while installation get from 1000 and the ones that we create from IV start from 1500. So, if we get the minimum uid from this function, we will not get the names of users that we create at the time of installation. I hard coded it as 1000 for now though a better alternative would be to keep track of the users that we create at the time of installation and get the rest of the users by fetching the min uid from config.py.